<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmIspisKuhinja</class>
 <widget class="QDialog" name="frmIspisKuhinja">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>214</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ispis kuhinja</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="1" colspan="2">
    <widget class="QLineEdit" name="txtRazmak">
     <property name="text">
      <string>3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Ispis razmak</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Stol br</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="QLineEdit" name="txtStolBr">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="btnIspis">
     <property name="text">
      <string>Ispis</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/ikone/ikone/button_ok.svg</normaloff>:/ikone/ikone/button_ok.svg</iconset>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="2">
    <widget class="QPushButton" name="btnIzlaz">
     <property name="text">
      <string>Izlaz</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/ikone/ikone/button_cancel.svg</normaloff>:/ikone/ikone/button_cancel.svg</iconset>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="chkIspisSaCijenama">
     <property name="text">
      <string>Sa cijenama</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="chkIspisNeIspisIspisane">
     <property name="text">
      <string>Ne ispisuj ispisane</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="chkIspisHrana">
     <property name="text">
      <string>Samo hrana</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="btnUpisIzmijene">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Upis izmijene</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/ikone/ikone/button_ok.svg</normaloff>:/ikone/ikone/button_ok.svg</iconset>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>chkIspisSaCijenama</tabstop>
  <tabstop>txtRazmak</tabstop>
  <tabstop>txtStolBr</tabstop>
  <tabstop>btnIspis</tabstop>
  <tabstop>btnIzlaz</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
